samples/client/petstore/spring-cloud-feign-without-url/src/main/java/org/openapitools/configuration/ClientConfiguration.java

```diff
@@ -21,7 +21,6 @@
 import org.springframework.context.annotation.Configuration;
 
 
-@Configuration
 public class ClientConfiguration {
```

Comment:
This still works when the `@Configuration` is removed?

Reply:
Yes because it’s in the @FeignClient Annotation. Then it’s scoped for this client(s)